python/py-rawpy/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy 0.17.2 v
+github.tarball_from archive
+
+checksums           rmd160  9606f685559195220c4d0557644d85780db00144 \
+                    sha256  31d76329c231131b606bda0c0066ae69e0ed50839cd2ccd8c0bec278c1bb7a19 \
+                    size    67235904
+
+python.versions     310
```

Comment:
Please set `python.pep517 yes` if that doesn't break the build.

Reply:
Just tried to enable it and it works.